### PR TITLE
Build python linux artifacts on Jenkins

### PR DESCRIPTION
**Pull request by jtattermusch**

### CODE REVIEW

**Comment by jtattermusch** [comment]
One problem is the 32bit docker still builds artifacts named x86_64  (they are probably 32bit in reality, but the architecture name is picked up incorrectly - we have a similar problem for ruby).

**Comment by soltanmm** [comment]
The target architecture in the egg name in Python is coming from an invocation of the `egg_info` command. I don't believe the problem (if we know for sure that there is one) is in the custom Python commands.

**Comment by jtattermusch** [comment]
Let's solve the 32bit package naming problem in a followup PR.
Can you please review the rest of this PR?

File: tools/dockerfile/grpc_artifact_linux_x64/Dockerfile [hunk 1]
```diff
@@ -58,6 +58,20 @@ RUN apt-get update && apt-get install -y \
   wget \
   zip && apt-get clean
 
+
+##################
```
Comment (ctiller) [inline tools/dockerfile/grpc_artifact_linux_x64/Dockerfile]: Are we better off separating Dockerfiles by language now?

Comment (jtattermusch) [inline tools/dockerfile/grpc_artifact_linux_x64/Dockerfile]: Yeah, maybe. @murgatroid99  didn't like the idea too much and for artifacts I'm fine both ways. We definitely need separate & lightweight docker image for testing the packages though (which is the case currently).

Also, this just got merged, so let's discuss this and address in a followup PR.

**Pull request merged**